CDP4Composition/ViewModels/ParameterRowControlViewModel.cs

```diff
@@ -181,7 +181,13 @@ private void UpdateProperties()
 
             this.Name = this.Parameter.ParameterType.Name;
             this.ShortName = this.Parameter.ParameterType.ShortName;
-            this.Value = $"{valueSet?.Published.FirstOrDefault()} [{this.Parameter.Scale.ShortName}]";
+            var actualvalue = valueSet?.ActualValue.FirstOrDefault() ?? "-";
```

Comment:
Why was this changed from published to actual?

Reply:
I thought I made a mistake by displaying the published Value, When I had a look at the row view model from the ElementDefinition tree, I thought the Actual Value need to be shown, I'll roll back then

Reply:
roll back and leave Published. In theory the row control should probably show both and make a distinction between the two as "Value" can be vague.